dipy/data/fetcher.py

```diff
@@ -1782,3 +1779,88 @@ def fetch_hcp(subjects,
                            "PipelineDescription": {'Name': 'hcp_pipeline'}})
 
     return data_files, pjoin(my_path, study)
+
+
+def fetch_hbn_preproc(subjects, path=None):
```

Comment:
Can you rename it `fetch_hbn`?   You can add the `preproc` information in the docstring. 

Also, workflow is failing because `hbn` != `hbn_preproc`